### DISCUSSION TITLE
Error when I wrap ChakraProvider in root file(Index.js)

### DISCUSSION THREAD

**Discussion by vishnu2193** [chakra-ui/chakra-ui#6174] [Q&A]
Error when I wrap ChakraProvider in root file(Index.js),because In my project I need to wrap global context api in root file and when I wrap ChakraProvider  in between them I am getting error.

Please provide solution how to setup ChakraProvider  in root file with Global context api.

<img width="960" alt="image" src="https://user-images.githubusercontent.com/81963203/172797928-69cf102f-fd61-4ad2-a879-99583cfbe1d3.png">
<img width="960" alt="2022-06-09_13h38_15" src="https://user-images.githubusercontent.com/81963203/172798124-bc976909-c0b5-4ae5-b823-f748a71251e5.png">

**Comment by aroyan** [comment]
Works fine on mine, check [my sandbox](https://codesandbox.io/s/xenodochial-mcclintock-ijimi0?file=/src/index.js)
I think you have to put dot `.` before `Provider` in your context provider and add a default value to it so it'll become like this

```jsx
ReactDOM.render(
  <StrictMode>
      <Cart.Provider value={anotherValue}>
          <HomeProducts.Provider value={"Some Value"}>
              <ChakraProvider>
                  <App />
              </ChakraProvider>
	  </HomeProducts.Provider>
      </Cart.Provider>
  </StrictMode>
)
```
